crash_test.mk

```diff
@@ -37,6 +39,11 @@ crash_test_with_txn: $(DB_STRESS_CMD)
 	$(CRASHTEST_MAKE) whitebox_crash_test_with_txn
 	$(CRASHTEST_MAKE) blackbox_crash_test_with_txn
 
+crash_test_with_optimistic_txn: $(DB_STRESS_CMD)
+# Do not parallelize
```

Comment:
Just wondering why (copied from pessimistic one and I assume the same, but not sure why)